libs/ui/packages/shared/src/consts/urls.ts

```diff
@@ -0,0 +1 @@
+export const CDN_URL = 'https://cdn.live.ledger.com';
```

Comment:
The idea behind this pr is remove providers icons library from react-ui. The icons will not be longer imported from here.

Reply:
I've reworked this PR. Removed all references of `ProviderIcon` from `@ledgerhq/react-ui`. Replaced them with a newly added `ProviderIcon` in the LLD / LLM app